app/main.py

```diff
@@ -2,5 +2,17 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    results = {}
+
+    def wrapper(*args) -> Callable:
+        results_item = (func, args)
+        if results_item in results:
+            print("Getting from cache")
+            return results[results_item]
```

Comment:
You are checking if the results_item is in the 'results' dictionary and if it is, you print a message and return the cached result. However, the 'print' statement is not necessary and should be removed. It's a part of the debugging process and should not be present in the final code.